### PR TITLE
Add AWS/EC2CapacityReservations to the services list

### PR DESCRIPTION
**Pull request by luismy**
## Summary
Add `AWS/EC2CapacityReservations` to the services list

## Context
See issue #1324

### CODE REVIEW

**Comment by luismy** [comment]
@cristiangreco would you mind reviewing this one? Not sure if I also need to include a test and/or update docs. Happy to do that if required

File: pkg/config/services.go [hunk 1]
```diff
@@ -334,6 +334,10 @@ var SupportedServices = serviceConfigs{
 			regexp.MustCompile("(?P<FleetRequestId>.*)"),
 		},
 	},
+	{
+		Namespace: "AWS/EC2CapacityReservations",
+		Alias:     "ec2CapacityReservations",
```
Comment (cristiangreco) [inline pkg/config/services.go:339]:
```suggestion
		Alias:     "ec2CapacityReservations",
		DimensionRegexps: []*regexp.Regexp{
				regexp.MustCompile(":capacity-reservation/(?P<CapacityReservationId>)$"),
		},
```

Comment (cristiangreco) [reply pkg/config/services.go:339]: I haven't tested this... do need to define a regexp here in order to capture tagged resources?

**Comment by luismy** [comment]
I've tested against my sandbox AWS account and you get all the reservation
ids if I remember correctly. I can add that for completeness as it will be
optional

Let me run a couple of tests against my account and I will let you know

**Comment by cristiangreco** [comment]
@luismy any update on this one?

File: pkg/config/services.go (hunk 1 above)
Comment (kgeckhart) [reply pkg/config/services.go:339]: According to https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html that's would match the format for a capacity-reservation resource
image

Comment (cristiangreco) [reply pkg/config/services.go:339]: Yep, I initially drafted it following that spec. I'll go ahead and merge the PR with this change included, we can always fix later if necessary.

**Pull request merged**